static/app/components/replays/replayContext.tsx

```diff
@@ -373,7 +373,7 @@ export function Provider({
           lineWidth: 2,
           strokeStyle: theme.purple200,
         },
-        plugins: organization.features.includes('session-replay-enable-canvas')
+        plugins: organization.features.includes('session-replay-enable-canvas-replayer')
```

Comment:
Can you split this into a new PR? We should avoid making frontend + backend changes in the same PR since they are not deployed simultaneously.